site/content/en/docs/enterprise/shapes-converter.md

```diff
@@ -0,0 +1,63 @@
+---
+title: 'Shapes converter'
+linkTitle: 'Shapes converter'
+weight: 4
+description: 'How to perform bulk actions on filtered shapes'
+---
+
+The shapes converter is a feature that enables bulk actions on filtered shapes. For example, it allows you to delete
+all filtered shapes or convert masks to polygons, among other functions.
```

Comment:
From this sentence I may conclude that shape converter allows me to delete filtered shapes

There is a difference between Shape converters and Annotations actions, that is not the same thing:

- Shape converters are subset of annotations actions, available for enteprise
- Annotation actions themselfes is open source feature